### DISCUSSION TITLE
How to only resolve part of ambiguites?

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#196] [Q&A]
Dear developers,
  Since there is something wrong in BeiDou's ambiguities resolution, how can I only resolve ambiguities of GPS and GALILEO but not resolve those of BDS when I use GPS+GALILEO+BDS at the same time? If possible, I'd like use GPS+GALILEO+BDS+GLONASS but not resolve ambiguities of GLONASS and BDS, what should I do to realize this?
Regards,
Yongzhao Fan

**Comment by tmayerguerr** [comment]
Hi Yongzhao Fan,
Unfortunately, that's not possible in groops at the moment. But that's a good point and we'll see how we can incorporate it. But it might take a while, because we have a lot of other things to do at the moment.
regards
Torsten
